databaseQueries/programs.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * name : programs.js
+ * author : Praveen
+ * created-date : 13-June-2024
+ * Description : program helper for DB interactions.
+ */
+
+// Dependencies 
+
+/**
+    * Programs
+    * @class
+*/
+
+
+
+module.exports= class Programs{
+   /**
+   * Programs Document.
+   * @method
+   * @name programDocuments
+   * @param {Array} [filterQuery = "all"] - solution ids.
```

Comment:
@praveenKDass check the description for filter query

Reply:
Changed @VISHNUDAS-tunerlabs